schedule/OHBM2021_schedule.csv

```diff
@@ -1,243 +1,271 @@
+Ensuring open science is accessible (OSR panel),"Stephen Klusza, Syreeta Nolan, Amanda Klinger, Alyssa Paparella",Stephanie Noble,20210623,50000,20210623,53000,,AmericasEurope,,,
+OSSIG fellows roundup 1 (OSR educational),"Han Zhang, Stephan Heunis, Martina G. Vilas, Marta Topor",,20210621,150000,20210621,152000,,AmericasEurope,,,
+OSSIG fellows roundup 1 (OSR educational),"Han Zhang, Stephan Heunis, Martina G. Vilas, Marta Topor",,20210623,230000,20210624,232000,,AmericasAsia,,,
+"Culture change towards more open, rigorous, and reproducible research (OSR educational)",Brian Nosek,,20210621,152000,20210621,154000,,AmericasEurope,,,
+"Culture change towards more open, rigorous, and reproducible research (OSR educational)",Brian Nosek,,20210623,232000,20210624,234000,,AmericasAsia,,,
+OSSIG fellows roundup 2 (OSR educational),"Sin Kim, Sidhant Chopra, Sina Mansour L., Jivesh Ramduny ",,20210621,230000,20210623,232000,,AmericasAsia,,,
+OSSIG fellows roundup 2 (OSR educational),"Sin Kim, Sidhant Chopra, Sina Mansour L., Jivesh Ramduny ",,20210624,150000,20210624,152000,,AmericasEurope,,,
+Building a user-driven database of open neuroscience projects (OSR educational),Matias Andina,,20210621,232000,20210623,234000,,AmericasAsia,,,
+Building a user-driven database of open neuroscience projects (OSR educational),Matias Andina,,20210624,152000,20210624,154000,,AmericasEurope,,,
+Empowering early career researchers to become open science leaders: the OLS open science ambassador program (OSR educational),Stephen Klusza,,20210622,150000,20210622,152000,,AmericasEurope,,,
+Empowering early career researchers to become open science leaders: the OLS open science ambassador program (OSR educational),Stephen Klusza,,20210624,230000,20210625,232000,,AmericasAsia,,,
+"ciftiTools is a package for reading, writing, visualizing and analyzing CIFTI and GIFTI files in R (OSR educational)",Amanda Mejia,,20210622,230000,20210623,232000,,AmericasAsia,,,
+"ciftiTools is a package for reading, writing, visualizing and analyzing CIFTI and GIFTI files in R (OSR educational)",Amanda Mejia,,20210625,150000,20210625,152000,,AmericasEurope,,,
+Coinstac Vaults for federated machine learning (OSR educational),Eric Verner,,20210622,232000,20210623,234000,,AmericasAsia,,,
+Coinstac Vaults for federated machine learning (OSR educational),Eric Verner,,20210625,152000,20210625,154000,,AmericasEurope,,,
+Failures in science and where to find them (OSR educational),Veronika Cheplygina,,20210623,150000,20210623,152000,,AmericasEurope,,,
+Failures in science and where to find them (OSR educational),Veronika Cheplygina,,20210625,230000,20210626,232000,,AmericasAsia,,,
+Data analysis pipelines: soluations to common problems (OSR educational),Marijn van Vliet,,20210623,152000,20210623,154000,,AmericasEurope,,,
+Data analysis pipelines: soluations to common problems (OSR educational),Marijn van Vliet,,20210625,232000,20210626,234000,,AmericasAsia,,,
```

Comment:
Are these the added events? If this is everything, it shouldn't be too difficult to integrate with the newest version. The main changes are: 

- time zone block name change -- easy to address with a find replace. 
- inclusion of `room` field: This is the full venue name, which is necessary for a specific sparkle feature
- might already be the case, but the required fields are now all except: moderator, description, uid, eventid. All others need to be there for the sparkle event requirements. 

Thanks! Sorry for the trouble, and let me know if you think these changes are doable. Otherwise, I'll take care of it.

Reply:
Also, worth noting at this stage there seems to be some strange character encoding in a few of the names.

Reply:
Hi Daniel,

I'll take care of the proposed changes an come back to you.

Reply:
I have no write access to the file, but below is the updated schedule:

Title,Speaker,Moderator,Date Start,Time Start,Date End,Time End,Location,TimeZoneBlock,Categories,UID,EVENTID

"The future of open tools/technologies (OSR LIVE panel)","Andre Maia Chagas, Lea Waller, Eilidh MacNicol, Christopher Mada",Oscar Esteban,20210622,170000,20210622,173000,opensciencesig,AmericasEurope,,,
"The future of open tools/technologies (OSR panel)","Andre Maia Chagas, Lea Waller, Eilidh MacNicol, Christopher Mada",Oscar Esteban,20210624,01000,20210624,01300,opensciencesig,AmericasAsia,,,
"Building a career in the age of open science (OSR LIVE panel)","Kirstie Whitaker, Vittorio Iacovella, Aki Nikoladis, Sal Kimmich",Russell A Poldrack,20210622,173000,20210622,173000,opensciencesig,AmericasEurope,,,
"Building a career in the age of open science (OSR panel)","Kirstie Whitaker, Vittorio Iacovella, Aki Nikoladis, Sal Kimmich",Russell A Poldrack,20210624,01000,20210624,01300,opensciencesig,AmericasAsia,,,
"Challenges in open science (OSR LIVE panel)","Olivia Guest, Mai-Lan Ho, Raghavendra Selvan, Danielle Kurtin",Aki Nikoladis,20210621,170000,20210621,173000,opensciencesig,AmericasEurope,,,
"Challenges in open science (OSR panel)","Olivia Guest, Mai-Lan Ho, Raghavendra Selvan, Danielle Kurtin",Aki Nikoladis,20210623,01000,20210623,01300,opensciencesig,AmericasAsia,,,
"Saving time and resources: open science and sustainability (OSR LIVE panel)","Dustin Moraczewski, Young J Woo, Winson Yang, Nikhil Bhagwat",Charlotte Rae,20210621,173000,20210621,180000,opensciencesig,AmericasEurope,,,
"Saving time and resources: open science and sustainability (OSR panel)","Dustin Moraczewski, Young J Woo, Winson Yang, Nikhil Bhagwat",Charlotte Rae,20210623,01300,20210623,02000,opensciencesig,AmericasAsia,,,
"Ensuring open science is accessible (OSR LIVE panel)","Stephen Klusza, Syreeta Nolan, Amanda Klinger, Alyssa Paparella",Stephanie Noble,20210622,01000,20210621,02000,opensciencesig,AmericasAsia,,,
"Ensuring open science is accessible (OSR panel)","Stephen Klusza, Syreeta Nolan, Amanda Klinger, Alyssa Paparella",Stephanie Noble,20210623,170000,20210623,180000,opensciencesig,AmericasEurope,,,
"OSSIG fellows roundup 1 (OSR educational)","Han Zhang, Stephan Heunis, Martina G Vilas, Marta Topor",Gabriel Gonzalez-Escamilla,20210621,150000,20210621,160000,opensciencesig,AmericasEurope,,,
"OSSIG fellows roundup 1 (OSR educational)","Han Zhang, Stephan Heunis, Martina G Vilas, Marta Topor",Eva Alonso Ortiz,20210623,230000,20210624,0,opensciencesig,AmericasAsia,,,
"Culture change towards more open, rigorous, and reproducible research (OSR educational)",,Brian Nosek,20210621,150000,20210621,160000,opensciencesig,AmericasEurope,,,
"Culture change towards more open, rigorous, and reproducible research (OSR educational)",,Brian Nosek,20210623,230000,20210624,0,opensciencesig,AmericasAsia,,,
"OSSIG fellows roundup 2 (OSR educational)","Sin Kim, Sidhant Chopra, Sina Mansour L, Jivesh Ramduny",Gabriel Gonzalez-Escamilla,20210621,230000,20210623,0,opensciencesig,AmericasAsia,,,
"OSSIG fellows roundup 2 (OSR educational)","Sin Kim, Sidhant Chopra, Sina Mansour L, Jivesh Ramduny",Eva Alonso Ortiz,20210624,150000,20210624,160000,opensciencesig,AmericasEurope,,,
"Building a user-driven database of open neuroscience projects (OSR educational)",,Matias Andina,20210621,230000,20210623,0,opensciencesig,AmericasAsia,,,
"Building a user-driven database of open neuroscience projects (OSR educational)",,Matias Andina,20210624,150000,20210624,160000,opensciencesig,AmericasEurope,,,
"Empowering early career researchers to become open science leaders: the OLS open science ambassador program (OSR educational)",,Stephen Klusza,20210622,150000,20210622,160000,opensciencesig,AmericasEurope,,,
"Empowering early career researchers to become open science leaders: the OLS open science ambassador program (OSR educational)",,Stephen Klusza,20210624,230000,20210625,0,opensciencesig,AmericasAsia,,,
"ciftiTools is a package for reading, writing, visualizing and analyzing CIFTI and GIFTI files in R (OSR educational)",,Amanda Mejia,20210622,230000,20210623,0,opensciencesig,AmericasAsia,,,
"ciftiTools is a package for reading, writing, visualizing and analyzing CIFTI and GIFTI files in R (OSR educational)",,Amanda Mejia,20210625,150000,20210625,160000,opensciencesig,AmericasEurope,,,
"Coinstac Vaults for federated machine learning (OSR educational)",,Eric Verner,20210622,230000,20210623,0,opensciencesig,AmericasAsia,,,
"Coinstac Vaults for federated machine learning (OSR educational)",,Eric Verner,20210625,150000,20210625,160000,opensciencesig,AmericasEurope,,,
"Failures in science and where to find them (OSR educational)",,Veronika Cheplygina,20210623,150000,20210623,160000,opensciencesig,AmericasEurope,,,
"Failures in science and where to find them (OSR educational)",,Veronika Cheplygina,20210625,230000,20210626,0,opensciencesig,AmericasAsia,,,
"Data analysis pipelines: soluations to common problems (OSR educational)",,Marijn van Vliet,20210623,150000,20210623,160000,opensciencesig,AmericasEurope,,,
"Data analysis pipelines: soluations to common problems (OSR educational)",,Marijn van Vliet,20210625,230000,20210626,0,opensciencesig,AmericasAsia,,,
"OSSIG social event",,OSSIG team,20210625,200000,20210625,220000,opensciencesig,AmericasAsia,,,
"BIDS town hall",,,20210621,200000,20210621,210000,opensciencesig,AmericasAsia,,,